test/test_pyfftw_numpy_interface.py

```diff
@@ -229,14 +229,6 @@ def _validate(self, array_type, test_shape, dtype,
 
         np_input_array = numpy.asarray(input_array)
 
-        # TODO why are long double inputs copied to double precision?
```

Comment:
This appears to have been necessary for numpy versions older than 1.10.  Starting from 1.10, each numpy.fft function always copies the input to double precision so that it no longer has to be done here.